src/components/interfaces/MOBILE_API.xml

```diff
@@ -3100,16 +3100,21 @@
         <param name="hdRadioEnable" type="Boolean" mandatory="false"  since="5.0">
             <description> True if the hd radio is on, false if the radio is off</description>
         </param>
-        <param name="availableHDs" type="Integer" minvalue="1" maxvalue="7" mandatory="false" since="5.0">
-            <description>number of HD sub-channels if available</description>
+        <param name="availableHDs" type="Integer" minvalue="0" maxvalue="7" mandatory="false" deprecated="true" since="5.1">
```

Comment:
need update since="5.1" to 5.2 or a proper mobile api version number

Reply:
@yang1070 Thanks. Fixed in 4d29747d10ac9d7d7d4294f7a7be4c35ee132c75